packages/nimble-components/src/rich-text/models/markdown-parser.ts

```diff
@@ -88,10 +100,40 @@ export class RichTextMarkdownParser {
                     // See: https://github.com/ni/nimble/issues/1527
                     excludes: '_',
                     toDOM(node) {
+                        const href = node.attrs.href as string;
+                        const currentMention = RichTextMarkdownParser.mentionConfigs?.find(
+                            mention => mention.isValidMentionHref(href)
+                        );
+                        const displayName = currentMention?.getDisplayName(href);
+
+                        if (currentMention && displayName) {
+                            return [
+                                currentMention.viewElement,
+                                {
+                                    'mention-href': href,
+                                    'mention-label': displayName,
+                                    'disable-editing': true
+                                }
+                            ];
+                        }
+
                         return [
                             anchorTag,
                             {
-                                href: node.attrs.href as Attr,
+                                /**
+                                 * Both mention and absolute link markdown share the autolink format in CommonMark flavor.
+                                 * However, absolute links without HTTPS/HTTP should render as plain text, while mentions are
+                                 * displayed as view elements.
+                                 *
+                                 * To achieve this, configuration elements matching already rendered absolute links in the rich
+                                 * text components should be converted to mention nodes. We should preserve markdown for
```

Comment:
This part doesn't seem right. By "configuration elements" do you mean `rich-text-mention-users` and `nimble-mapping-user`? Those don't know anything about the markdown content, so they aren't converted to mentions.

Reply:
I meant the "configuration element" is `rich-text-mention-users` and updated the same in detail there. Also, I meant the URL rendered in the components should be converted to mention neither the configuration element nor the markdown string.